client/layout/masterbar/logged-in.jsx

```diff
@@ -63,6 +63,7 @@ export default React.createClass( {
 		return (
 			<Masterbar>
 				<Stats
+					tip="my-sites"
```

Comment:
`name`? :trollface: 

Coming up with something both concise _and_ not super confusing for outsiders seems hard. Another option is to use something very searchable, like `stepName`, `guidesToursName`… Barf.